src/api.js

```diff
@@ -36,4 +38,16 @@ apiRouter.get("/movies/:id/reviews", async (req, res) => {
 
 });
 
+apiRouter.get("/screenings", async (req,res)=> {
+	let data;
+	let filteredScreenings;
+	try {
+		data = await loadScreenings();
```

Comment:
I would recommend using `const data` here, instead of declaring the variable up top and then assigning it here.